### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rainleander**
editorial changes with input from: Lisa, Rain, Amruta, Meagan, and kena

### CODE REVIEW

**Review by jordanlewis — COMMENTED** [review]
Looks good except for one small comment!

**Review by knz — COMMENTED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/66264)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @rainleander)

___
*[README.md, line 61 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/66264#-MblQ2bd-TdnP01l9xmq:-MblQ2bd-TdnP01l9xmr:b-v5by5h) ([raw file](https://github.com/cockroachdb/cockroach/blob/2edfb03e439ad93127d988cad8ba6f4d63cc6068/README.md#L61)):*
> ```Markdown
> 
> - [CockroachCloud](https://www.cockroachlabs.com/docs/cockroachcloud/quickstart) - Steps to deploy a [free CockroachCloud cluster](https://cockroachlabs.cloud/signup?referralId=githubquickstart) on public Cloud platforms.
> - [Manual](https://www.cockroachlabs.com/docs/stable/manual-deployment.html) - Steps to deploy a CockroachDB cluster manually on multiple machines.
> ```

Are we sure we want to remove the manual steps entirely? Maybe couple this with manual install steps?

File: README.md [hunk 1]
```diff
@@ -37,63 +39,51 @@ https://www.cockroachlabs.com/docs/stable/architecture/overview.html).
 
 For guidance on installation, development, deployment, and administration, see our [User Documentation](https://cockroachlabs.com/docs/stable/).
 
-## Quickstart
+## Starting with CockroachCloud
 
-1. [Install CockroachDB](https://www.cockroachlabs.com/docs/stable/install-cockroachdb.html).
-2. [Start a local cluster](https://www.cockroachlabs.com/docs/stable/start-a-local-cluster.html)
-   and talk to it via the [built-in SQL client](https://www.cockroachlabs.com/docs/stable/use-the-built-in-sql-client.html).
+We can run CockroachDB for you, so you don't have to run your own server.
```
[the remark this answers is not in the document]
Comment (rainleander) [reply README.md:44]: Agreed.

**Comment by rainleander** [comment]
> > Reviewed 1 of 1 files at r2.
> > _[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/66264)_ status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @rainleander)
> 
> _[README.md, line 61 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/66264#-MblQ2bd-TdnP01l9xmq:-MblQ2bd-TdnP01l9xmr:b-v5by5h) ([raw file](https://github.com/cockroachdb/cockroach/blob/2edfb03e439ad93127d988cad8ba6f4d63cc6068/README.md#L61)):_
> 
> > ```gfm
> > - [CockroachCloud](https://www.cockroachlabs.com/docs/cockroachcloud/quickstart) - Steps to deploy a [free CockroachCloud cluster](https://cockroachlabs.cloud/signup?referralId=githubquickstart) on public Cloud platforms.
> > - [Manual](https://www.cockroachlabs.com/docs/stable/manual-deployment.html) - Steps to deploy a CockroachDB cluster manually on multiple machines.
> > ```
> 
> Are we sure we want to remove the manual steps entirely? Maybe couple this with manual install steps?

Good catch. I think the idea was that we were going to fold this into the getting started with cockroachdb section, but now it's missing from both. I'm going to put it in the original position until I can figure out where the request for migration came from... Thx @knz

**Review by knz — APPROVED** [review]
Changes LGTM,
but please squash the commits and add a release note to the commit message

cheers

> Reviewed 1 of 1 files at r4.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/66264#-:-MbqGS_aEiSwvZUvGlYT:b5431dn)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @jordanlewis)

**Comment by rainleander** [comment]
Release note: Audit and update wording to be more inclusive.

Squashing commits today (and hoping @jordanlewis doesn't have anything to add / change)

**Review by knz — APPROVED** [review]
:lgtm:

let's :shipit: 

bors r=jordanlewis,knz

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/66264#-:-McDgGsE20pTbPXnqZzq:bjpvvsf)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @jordanlewis)

**Pull request merged**